File: 2520-count-the-digits-that-divide-a-number/2520-count-the-digits-that-divide-a-number.py
class Solution(object):
    def countDigits(self, num):
        """
        :type num: int
        :rtype: int
        """
        stri = str(num)
        result = 0
        for single in stri:
            if num % int(single) == 0:
                result += 1
        return result
        

File: 2520-count-the-digits-that-divide-a-number/2520-count-the-digits-that-divide-a-number.java
public class Solution {
    public int countDigits(int num) {
        String stri = Integer.toString(num);
        int result = 0;
        for (char single : stri.toCharArray()) {
            if (num % Character.getNumericValue(single) == 0) {
                result += 1;
            }  
        }
        return result;
        
    }
}
